Documentation/contributing/development/hive.rst

```diff
@@ -403,12 +403,25 @@ returns a cell that "provides" the parsed configuration to the application:
 
     type MyConfig struct {
         MyOption string
+
+        SliceOption []string
+        MapOption map[string]string
     }
 
     func (def MyConfig) Flags(flags *pflag.FlagSet) {
         // Register the "my-option" flag. This matched against the MyOption field
         // by removing any dashes and doing case insensitive comparison.
         flags.String("my-option", def.MyOption, "My config option")
+
+        // Flags representing complex types such as slices and maps are supported.
```

Comment:
If we can't fully use active voice, I'd like less distance between the subject and its most important qualifiers.
```suggestion
        // Flags are supported for representing complex types such as slices and maps.
```